code/addons/links/template/stories/parameters.stories.ts

```diff
@@ -0,0 +1,44 @@
+import globalThis from 'global';
+import { linkTo } from '@storybook/addon-links';
+
+export default {
+  component: globalThis.Components.Button,
+  args: {
+    children: 'Click Me!',
+  },
+  parameters: {
+    chromatic: { disable: true },
+  },
+};
+
+export const ID = {
+  args: {
+    onClick: linkTo('addons-links-parameters--basic'),
```

Comment:
Why are these called `parameters.stories.ts`? Wouldn't `linkTo.stories.ts` or similar be more descriptive?